File: WebApplication/src/modules/student/dtos/person-lot.dtos/person-lot.dto.ts
import { LotState } from "../../enums/lot-state.enum";
import { PersonBasicDto } from "./person-basic.dtos/person-basic.dto";
import { PersonDoctoralDto } from "./person-doctoral.dtos/person-doctoral.dto";
import { PersonSecondaryDto } from "./person-secondary.dtos/person-secondary.dto";
import { PersonStudentDto } from "./person-student.dtos/person-student.dto";

export class PersonLotDto {
  uan: string;
  lotState: LotState;
  createDate: Date;

  personBasicDto: PersonBasicDto = new PersonBasicDto();
  personSecondaryDto: PersonSecondaryDto = new PersonSecondaryDto();
  personStudentsDto: PersonStudentDto[] = [];
  personDoctoralsDto: PersonDoctoralDto[] = [];
}

File: StudentCard.Application/Students/Dtos/PersonLotDtos/PersonLotDto.cs
using StudentCard.Application.Students.Dtos.PersonLotDtos.PersonBasicDtos;
using StudentCard.Application.Students.Dtos.PersonLotDtos.PersonDoctoralDtos;
using StudentCard.Application.Students.Dtos.PersonLotDtos.PersonSecondaryDtos;
using StudentCard.Application.Students.Dtos.PersonLotDtos.PersonStudentDtos;
using StudentCard.Data.Rdpzsd.Enums;
using System;
using System.Collections.Generic;

namespace StudentCard.Application.Students.Dtos.PersonLotDtos
{
    public class PersonLotDto
    {
        public string Uan { get; set; }
        public LotState State { get; set; }
        public DateTime CreateDate { get; set; }

        public PersonBasicDto PersonBasicDto { get; set; }
        public PersonSecondaryDto PersonSecondaryDto { get; set; }
        public List<PersonStudentDto> PersonStudentsDto { get; set; } = new List<PersonStudentDto>();
        public List<PersonDoctoralDto> PersonDoctoralsDto { get; set; } = new List<PersonDoctoralDto>();
    }
}
